AndelaOSP/wire-api
# #163729655 Update test config variables

#### What does this PR do?
- Ensures the test variables in the config are obtained from the environment.

#### Description of Task to be completed?
- Configure the config file to obtain critical variables from the environment.

#### How should this be manually tested?
- Clone the repo `https://github.com/AndelaOSP/wire-api.git`
- Cd into the `wire-api` and then the `src` directory.
- Add a `.env` file as shown in the `env.example` file.
- Run the test using `$ yarn test`. The tests should run.

#### Any background context you want to provide?
- Previously you had to export the environment in the terminal before running the tests.

#### What are the relevant pivotal tracker stories?
[#163729655](https://www.pivotaltracker.com/story/show/163729655)
#### Screenshots (if appropriate)
N/A
